.travis.yml

```diff
@@ -46,7 +46,7 @@ env:
   - PACKAGE=surelog
   - PACKAGE=sv-parser
   - PACKAGE=verible             USE_SYSTEM_GCC_VERSION="8"
-  - PACKAGE=uhdm-integration
```

Comment:
Can we make GCC_VERSION="8" the minimum?

Reply:
I set it to 7 because we have GCC 7 in our CI for `uhdm-integration`. I wanted to avoid any unnecessary problems du to incompatibilities.